### PR TITLE
Documentation folder and Eclipse IDE tutorial

### PR DESCRIPTION
**Pull request by devtodev**
Pull request to issue #77

### CODE REVIEW

File: documentation/eclipse-ide.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Get [AnyLedger-wallet](https://github.com/AnyLedger/anyledger-wallet) source code. ```git clone https://github.com/AnyLedger/anyledger-wallet.git```
+
+Check the [README.md](https://github.com/AnyLedger/anyledger-wallet/blob/master/README.md) of the project and ensure you have all you need to compile it.
+
+Download [GNU MCU Eclipse IDE for C/C++ Developers](https://github.com/gnu-mcu-eclipse/org.eclipse.epp.packages/releases/), open Eclipse and follow this instructions:
+
+* Go to the Eclipse menu → Help → Eclipse Marketplace…, find GNU MCU Eclipse and install
+* If anything go bad in the Eclipse instalation, this [link](https://gnu-mcu-eclipse.github.io/plugins/install/) will give you more details.
+* To get a more comfortable IDE experience: Eclipse Menu → Window →  Preferences, General →  Appearance, Theme: Dark.
+* Configure the path to the pyOCD GDB server by opening the menu Window->Preferences, navigating to MCU, and setting the Global pyOCD Path, in linux should be /usr/bin, to check it run: ```$ find / -name pyocd-gdbserver```
```
Comment (pcppcp) [inline documentation/eclipse-ide.md]: whoa, find on `/`? how about `which` or `whereis`?

Comment (devtodev) [reply documentation/eclipse-ide.md]: locate command could works too :D

Comment (devtodev) [reply documentation/eclipse-ide.md]: locate command could works too :D

Comment (devtodev) [reply documentation/eclipse-ide.md]: locate command could works too :D

**Pull request merged**